docs/man_pages/project/creation/create.md

```diff
@@ -3,26 +3,32 @@ create
 
 Usage | Synopsis
 ---|---
-General | `$ tns create <App Name> [--path <Directory>] [--appid <App ID>] [--copy-from <Directory>]`
+General | `$ tns create <App Name> [--path <Directory>] [--appid <App ID>] [--copy-from <Directory>] [--template <Template>]`
 
 Creates a new project for native development with NativeScript from the default template or from an existing NativeScript project.
 
 ### Options
 * `--path` - Specifies the directory where you want to create the project, if different from the current directory. The directory must be empty.
-* `--appid` - Sets the application identifier for your project. 
+* `--appid` - Sets the application identifier for your project.
 * `--copy-from` - Specifies a directory which contains an existing NativeScript project. If not set, the NativeScript CLI creates the project from the default hello-world template.
+* `--template` - Sets the source template for the project. The value can be anything that you can `npm install` - npm package, local path, .tgz, GitHub URL. The package will be used as `app` directory of the new application.
+<% if(isHtml) { %>
+If `--template typescript` or `--template tsc` is specified, the default TypeScript template (`tns-template-hello-world-ts`) will be used.
```

Comment:
I think we should present this in the CLI help as well and not limit it to the HTML one.